### DISCUSSION TITLE
PI Zero W 0

### DISCUSSION THREAD

**Discussion by spacemishka** [aluminum-ice/pwnagotchi#67] [General]
Hi,

is this still working with the old Zero W?

thanks
Peter

**Comment by wpa-2** [comment]
> Hi,
> 
> is this still working with the old Zero W?
> 
> thanks Peter

Sadly not but this does 
https://github.com/wpa-2/pwnagotchi/releases/tag/Fix

**Reply by spacemishka** [reply]
thank you
